content/modules/r/pages/paste-and-paste0.adoc

```diff
@@ -0,0 +1,57 @@
+= `paste` and `paste0`
+
+`paste` is a useful function to "concatenate vectors after converting to character."
+
+`paste0` is a shorthand function where the sep argument is "".
```

Comment:
`paste0` is similar to the `paste` function except its `sep` argument is "". That means it will concatenate the vectors without any delimiting character (e.g., space, comma, semi-colon).